src/Microsoft.AspNet.Mvc.Core/Controllers/FilterActionInvoker.cs

```diff
@@ -241,6 +241,17 @@ private IFilterMetadata[] GetFilters()
                 _filterProviders[i].OnProvidersExecuted(context);
             }
 
+            if (_diagnosticSource.IsEnabled("Microsoft.AspNet.Mvc.RegisteredFilters"))
+            {
+                _diagnosticSource.Write(
+                    "Microsoft.AspNet.Mvc.RegisteredFilters",
+                    new
+                    {
+                        actionDescriptor = ActionContext.ActionDescriptor,
+                        filterProviderContext = context
+                    });
+            }
```

Comment:
At this point the `FilterProviderContext` contains a list of `FIlterItem` which provides the FilterDescriptors and Filters